CodeReviewChallenge.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+​
+public class Class1 {
+    public final String SPECIAL_CHARACTER = "a";
+​
+    public List<String> method(String paragraph) {
+        List<String> shortestWords = new ArrayList<>();
+        String[] sentences =   paragraph.split(".");
+​
+        for (int i=0;i<sentences.length;i++) {
+            String[] words = sentences[i].split(" ");
+            String shortestWord = words[0];
+​
+            for (int j = 0; j < words.length; j++) {
+                if (words[j].length() < shortestWord.length()) {
+                    if (words[j].contains("a")) {
+                        shortestWord = words[j];
+                    }
+                }
+            }
+​
+            shortestWords.add(shortestWord);
+        }
+​
+​
+​
+        return shortestWords;
+    }
+​
+    public String findShortestWordWithA(String paragraph) {
```

Comment:
the method name should be more generalized e.g. findShortestWordWithSpecialCharacter(String paragraph), since if we change the special character the method wouldn't make any sense